launchable/commands/record/build.py

```diff
@@ -160,7 +160,9 @@ def calc_branch_name(self):
 
                 if len(refs) > 0:
                     # e.g) ed6de84bde58d51deebe90e01ddfa5fa78899b1c refs/heads/branch-name
-                    self.branch = refs[0].split("/")[-1]
+                    match = re.search('[a-f0-9]{40} refs/heads/(.*)', refs[0])
```

Comment:
Do we have a case that commit SHA is not something like `ed6de84bde58d51deebe90e01ddfa5fa78899b1c refs/heads/branch-name`?

Reply:
Just in case, I've added like this https://github.com/launchableinc/cli/pull/765/files#diff-dbb014559b2adb369474bcb0dde7d795e50930c4b604a3d94a6a53961159d845R167.